UPGRADE-4.2.md

```diff
@@ -216,4 +216,6 @@ Validator
  * The component is now decoupled from `symfony/translation` and uses `Symfony\Contracts\Translation\TranslatorInterface` instead
  * The `ValidatorBuilderInterface` has been deprecated and `ValidatorBuilder` made final
  * Deprecated validating instances of `\DateTimeInterface` in `DateTimeValidator`, `DateValidator` and `TimeValidator`. Use `Type` instead or remove the constraint if the underlying model is type hinted to `\DateTimeInterface` already.
- * Using the `Bic` constraint without `symfony/intl` is deprecated
+ * Using the `Bic`, `Country`, `Currency`, `Language` and `Locale` constraints without `symfony/intl` is deprecated
```

Comment:
missing final dots here and below (same below for other upgrade files)

Reply:
i think we only do that for multiple sequences...